apps/api/src/app/rate-limiting/usecases/get-default-api-rate-limits/get-default-api-rate-limits.usecase.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@nestjs/common';
+import {
+  ApiRateLimitCategoryTypeEnum,
+  ApiRateLimitEnvVarFormat,
+  ApiServiceLevelTypeEnum,
+  DEFAULT_API_RATE_LIMITS,
+  IServiceApiRateLimits,
+} from '@novu/shared';
+
+@Injectable()
+export class GetDefaultApiRateLimits {
```

Comment:
A use-case to merge the default values stored in code with the values configured via environment variables. This provides:
* Simple UX to fallback to hard-coded values
* Flexibility to define custom rate limits for self-hosted customers or Novu-hosted via environment vars

In the future, we may also support the ability for customers to resolves the default API rate limit values from an external source, such as a 3rd-party API.